### PR TITLE
feat: k8s example deployment with some goodies

### PR DESCRIPTION
**Pull request by ryancurrah**
If the FSM and CLF configs change chatto will auto-reload them. Set consistent env var names. Added an example deployment. Allow setting debug through env var.

Resolves https://github.com/jaimeteb/chatto/issues/35

### CODE REVIEW

**Comment by jaimeteb** [comment]
Looking great overall. Let me give it a more thorough check tomorrow!

**Comment by jaimeteb** [comment]
@ryancurrah love it, the auto-reload feature is great!
I only added resource limits to the container and removed the ASCII art smiley.

**Comment by ryancurrah** [comment]
Cool this needs to be updated now that the other PR has been merged. I will do that tonight.

**Comment by jaimeteb** [comment]
> Cool this needs to be updated now that the other PR has been merged. I will do that tonight.

Awesome! I've updated the k8s deployment already.

File: deploy/kubernetes/chatto.yml [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: chatto-config-files
+  namespace: default
+data:
+  fsm.yml: |
+    transitions:
+      - from:
+          - "question"
```
Comment (ryancurrah) [inline deploy/kubernetes/chatto.yml:39]:
```
- "question"
```

Should be

```
- "initial"
```

**Review by ryancurrah — COMMENTED** [review]
> > Cool this needs to be updated now that the other PR has been merged. I will do that tonight.
> 
> Awesome! I've updated the k8s deployment already.

Nice. Just need to replace `question` with `initial` now in the `fsm.yml` file.

**Comment by jaimeteb** [comment]
Done.

**Comment by ryancurrah** [comment]
Looks good now. Merge it! 🚀

**Pull request merged**